### PR TITLE
Update Italian.yml

### PR DESCRIPTION
**Pull request by giucarny**
Removed 'Città di Ho Chi Minh' and added 'Saigon' (#47).

### CODE REVIEW

File: dict/italian.yml [hunk 1]
```diff
@@ -1,291 +1,291 @@
...
+    'DJ': [Gibuti, gibutian*]
+    'ER': [Eritrea, eritre*, Asmara]
+    'ET': [Etiopia, etiop*, abissin*, Addis Abeba]
+    'KE': [Kenya, keniot*, kenyot*, kenian*, kenyan*, Nairobi]
+    'KM': [Comore, comorian*, Moroni]
+    'MG': [Madagascar, malgasc*, Antananarivo]
+    'MU': [Mauritius, maurizian*, Port Louis]
+    'MW': [Malawi, Malawian*, Lilongwe]
+    'MZ': [Mozambico, mozambican*, Maputo]
+    'RE': [Riunione, Saint-Denis]                                                              #inhabitants' Italian official name is missing
```
Comment (koheiw) [inline dict/italian.yml:15]: Can you explain what is "inhabitant's official name"?

**Review by koheiw — CHANGES_REQUESTED** [review]
Let's delete the comments on the official names because it is not only about Italian.

**Comment by giucarny** [comment]
https://github.com/koheiw/newsmap/pull/50#issuecomment-1036912070
Next time I will follow the workflow suggested by you. In the meantime, I believe that this PR could be merged, otherwise let me know and I'll create another one.

**Review by koheiw — APPROVED** [review]

**Comment by koheiw** [comment]
@giucarny thanks. I will remove the comments myself.

**Pull request merged**